### PR TITLE
Add Windows.10.Amd64.Server20H1.Open helix queue

### PR DESCRIPTION
**Pull request by Tratcher**
Let's see how well we work on new windows versions...

### CODE REVIEW

File: .azure/pipelines/helix-matrix.yml [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # We only want to run full helix matrix on master
-pr: none
+# Run PR validation on all branches
+pr:
+  autoCancel: true
+  branches:
```
Comment (Tratcher) [inline .azure/pipelines/helix-matrix.yml:5]: TODO: Revert

**Review by dougbu — APPROVED** [review]
First commit works for me. Careful w/ "green" requirements though -- failures in the `aspnetcore-helix-matrix` pipeline won't prevent merging and `blocked` labels are just a hint to the user 😺

**Comment by Tratcher** [comment]
Hmm, Windows.10.Amd64.Server20H1.Open timed out.

**Comment by Tratcher** [comment]
Skipped one set of tests:
https://github.com/dotnet/aspnetcore/issues/25768

**Comment by Tratcher** [comment]
Windows.10.Amd64.Server20H1.Open is now passing with the one skipped diagnostics project. I've reverted the change to run the full helix pass on PRs. This should be ready to merge after a fresh CI run.

**Comment by Tratcher** [comment]
@Pilchie Please approve/merge this test change for rc2.

**Comment by Pilchie** [comment]
I'm going to force merge this, given that it shouldn't affect these tests.

**Pull request merged**